README.md

```diff
@@ -234,6 +238,29 @@ Note the presence of a `default` which will catch any extension that is not list
 ]
 ```
 
+#### Acorn Plugins
+[Acorn](https://github.com/acornjs/acorn) supports plugins, which can be enabled by importing their injector functions and then applying them to the acorn instance:
+```javascript
+import acorn from 'acorn';
+import injectAcornStaticClassPropertyInitializer from 'acorn-static-class-property-initializer/inject';
+
+injectAcornStaticClassPropertyInitializer(acorn);
+```
+To support these plugins in this project, there is an `injectors` option on the `acorn` configuration key. To add additional plugins to the `JavascriptLexer` oor `JsxLexer`, the configuration would look something like this:
```

Comment:
I think it would gain clarity if we explicitly said that people need to install the dependency. Also I would document how to get to a sensible default setup, maybe list the main 3-4 plugins and what they do. It would make people's life easier.

Reply:
I like this idea.